perun-base/src/main/resources/perun-roles.yml

```diff
@@ -8181,13 +8186,33 @@ perun_roles_management:
       - PERUNADMIN:
```

Comment:
Theoretically, PERUNADMINBA should be able to set PERUNADMIN, no?

Reply:
I think this is not what we want because PERUNADMINBA is supposed to be just an internal role, which will be added manually through the database.

Reply:
I know that, but as an internal "super user" account, I don't see why it shouldn't be allowed to set PERUNADMIN. But I guess it's not a big deal.